CHANGELOG.asciidoc

```diff
@@ -46,6 +46,8 @@ See the <<upgrade-to-v4>> guide.
 [float]
 ===== Bug fixes
 
+* Fix path resolution for requests that contain invalid characters in its
```

Comment:
This should be added to a new ` ==== Unreleased` section. See template for the new section at the top of the CHANGELOG file.